packages/components/ng-ovh-payment-method/src/payment-method-type.class.js

```diff
@@ -96,4 +99,42 @@ export default class OvhPaymentMethodType {
       TYPE_INTEGRATION_ENUM.IN_CONTEXT,
     ].includes(this.integration);
   }
+
+  /**
+   * Determine if payment method type is a Bank Account.
+   *
+   * @return {Boolean}
+   */
+  isBankAccount() {
+    return this.paymentType === PAYMENT_METHOD_TYPE_ENUM.BANK_ACCOUNT;
```

Comment:
Why do this needs to be duplicate ? Isn't there a parent class ?

Reply:
Mmmmh, can be done but we talk about two different concept here:
- payment methods which have payment method type ;
- payment method types which are used to add a new payment method of given type.

So I don't think it's a good solution to have a mix of concepts. 
Maybe a static method to centralize it or maybe define `paymentType` attribute of `OvhPaymentMethod` instance as an instance of `OvhPaymentMethodType`.

Reply:
Yes Maybe `OvhPaymentMethod.type` should be an instance of OvhPaymentMethodType